INFO     core.jenkins_utils:jenkins_utils.py:210 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:92 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 133
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:61 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:61 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:61 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:61 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:61 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:61 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 136.0.7103.92 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 136.0.7103.92 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 136.0.7103.92
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 136.0.7103.113
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 12781 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:60183
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "GET /session/278e1a045b200f5903aa5f4f0ec81376/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/element/f.05B18E34BA489193F909D4680D0EFE45.d.FCE59EF9870ADFFC14C98FC64A35796E.e.1/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/element/f.05B18E34BA489193F909D4680D0EFE45.d.FCE59EF9870ADFFC14C98FC64A35796E.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/element/f.05B18E34BA489193F909D4680D0EFE45.d.FCE59EF9870ADFFC14C98FC64A35796E.e.9/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "GET /session/278e1a045b200f5903aa5f4f0ec81376/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "GET /session/278e1a045b200f5903aa5f4f0ec81376/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 156
DEBUG    core.jenkins_utils:jenkins_utils.py:31 update_crumb: d78233a9d2983b61e6228e62c429071c2104f30174d1374fe4c8f37249dcae39
DEBUG    LoginPage:login_page.py:32 login crumb: d78233a9d2983b61e6228e62c429071c2104f30174d1374fe4c8f37249dcae39
DEBUG    MainPage:ui_element.py:81 Navigating to NewItemPage with click on (('link text', 'New Item'),)
DEBUG    MainPage:ui_element.py:28 waiting for element_to_be_clickable(('link text', 'New Item'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "GET /session/278e1a045b200f5903aa5f4f0ec81376/element/f.05B18E34BA489193F909D4680D0EFE45.d.3411A5D047C38A1055644D8CC4885F03.e.47/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/element/f.05B18E34BA489193F909D4680D0EFE45.d.3411A5D047C38A1055644D8CC4885F03.e.47/click HTTP/1.1" 200 0
DEBUG    NewItemPage:base_page.py:22 Waiting for page content to load: http://localhost:8080/view/all/newJob
DEBUG    NewItemPage:ui_element.py:28 waiting for presence_of_element_located(('xpath', "//h1[text()='New Item']"),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/element HTTP/1.1" 200 0
DEBUG    NewItemPage:base_page.py:25 Page content loaded: http://localhost:8080/view/all/newJob
DEBUG    NewItemPage:ui_element.py:28 waiting for presence_of_element_located(('css selector', '#ok-button'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/element/f.05B18E34BA489193F909D4680D0EFE45.d.36F7980A7F105757D55A0AEACF7390FC.e.98/click HTTP/1.1" 200 0
DEBUG    NewItemPage:ui_element.py:28 waiting for presence_of_element_located(('id', 'itemname-required'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "POST /session/278e1a045b200f5903aa5f4f0ec81376/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "GET /session/278e1a045b200f5903aa5f4f0ec81376/element/f.05B18E34BA489193F909D4680D0EFE45.d.36F7980A7F105757D55A0AEACF7390FC.e.120/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60183 "DELETE /session/278e1a045b200f5903aa5f4f0ec81376 HTTP/1.1" 200 0